.github/workflows/release.yaml

```diff
@@ -6,14 +6,29 @@ permissions: write-all
 env:
   ContainerRegistry: "ghcr.io"
   ContainerRegistryRepo: "ghcr.io/eclipse-symphony"
-
+  BOT_USER_NAME: eclipse-symphoy-bot
+  BOT_EMAIL_ID: symphony-bot@eclipse.org
+  REGISTRY: ghcr.io
```

Comment:
you already have defined a ContainerRegistry variable above and dont use this one. So I would clean this up. When you continue using your existing vars, you dont need to include the 2 one here afaict.

Reply:
You are right. Cleaned this.